src/map/elemental.c

```diff
@@ -97,13 +97,13 @@ int elemental_create(struct map_session_data *sd, int class_, unsigned int lifet
 	summon_level = db->status.size+1; // summon level
 
 	//[(Caster's Max HP/ 3 ) + (Caster's INT x 10 )+ (Caster's Job Level x 20 )] x [(Elemental Summon Level + 2) / 3]
-	ele.hp = ele.max_hp = (sd->battle_status.max_hp/3 + sd->battle_status.int_*10 + sd->status.job_level) * ((summon_level + 2) / 3);
+	ele.hp = ele.max_hp = ((sd->battle_status.max_hp / 3) + (sd->battle_status.int_ * 10) + (sd->status.job_level * 20)) * ((summon_level + 2) / 3);
```

Comment:
@Smokexyz, as @Asheraf said, those commits that you mention don't touch the formula or the comment, they're just fixes to the character encoding and/or variable names. The formula and the conflicting comment were both written in commit 97166dfb. Which is one of those absurd 1800-lines mega-commits that people used to do before we started enforcing a code review policy.